config/config.exs

```diff
@@ -0,0 +1,3 @@
+use Mix.Config
```

Comment:
```suggestion
import Config
```
Mix.Config is deprecated. Unless we are testing under a old version of elixir, may make sense to keep using it.